release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1937,7 +1938,7 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 
 * Previously, on the *Observe -> Alerting -> Silences* tab, the `DateTime` component changed the ordering of an event and its value. Because of this issue, you could not edit the `until` parameter for a silent alert in either the *Developer* or the *Administrator* perspective. With this release, a fix means to the `DateTime` component means that you can now edit the `until` parameter for a silent alert. (link:https://issues.redhat.com/browse/OCPBUGS-46021[*OCPBUGS-46021*])
 
-* Previously, when using the *Developer* perspective with custom editors, clicking the `n` key caused the *Namespace* menu unexpectedly opened. The issue happened because the keyboard shortcut did not account for custom editors. With this release, the *Namespace* menu accounts for custom editors and does not open if you type 'n' key. (link:https://issues.redhat.com/browse/OCPBUGS-38775[*OCPBUGS-38775*])
+* Previously, when using the *Developer* perspective with custom editors, clicking the `n` key caused the *Namespace* menu unexpectedly opened. The issue happened because the keyboard shortcut did not account for custom editors. With this release, the *Namespace* menu accounts for custom editors and does not open if you type the `n key. (link:https://issues.redhat.com/browse/OCPBUGS-38775[*OCPBUGS-38775*])
```

Comment:
```suggestion
* Previously, when using the *Developer* perspective with custom editors, clicking the `n` key caused the *Namespace* menu unexpectedly opened. The issue happened because the keyboard shortcut did not account for custom editors. With this release, the *Namespace* menu accounts for custom editors and does not open if you type the `n` key. (link:https://issues.redhat.com/browse/OCPBUGS-38775[*OCPBUGS-38775*])
```